services/core-web/src/tests/components/dashboard/majorProjectHomePage/MajorProjectSearch.spec.js

```diff
@@ -1,7 +1,8 @@
 import React from "react";
-import { shallow } from "enzyme";
 import MajorProjectSearch from "@/components/dashboard/majorProjectHomePage/MajorProjectSearch";
 import * as MOCK from "@/tests/mocks/dataMocks";
```

Comment:
may as well update this import to be from @mds/common. It has more complete data and will give more useful results. Same for the other test below.

Reply:
The dataMocks has been changed to be from @mds/common here too.